### PR TITLE
chore: Remove repo upgrade test

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

This PR removes a bunch of unused scripts, and the repository upgrade test, wich is no longer useful to us (we cannot currently add meaningful new upgrade plugins, and the old ones we don't need to test over and over again; also the data is no longer a good approximation of real production data).

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [ ] feat: represents new features
- [ ] fix: represents bug fixes
- [ ] perf: performance improvements
- [ ] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)
- [ ] deprecated: Deprecation warning (ideally referencing a migration guide)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### CODE REVIEW

**Comment by BalduinLandolt** [comment]
The tests cannot complete because it's waiting for the mandatory "Test repository upgrade" job. I'll remove that from branch protection once I have approval for the PR.

**Review by seakayone — APPROVED** [review]

**Review by mpro7 — APPROVED** [review]
👏

**Pull request merged**
